presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/test/RuleTester.java

```diff
@@ -65,9 +66,42 @@ public RuleAssert assertThat(Rule rule)
         return new RuleAssert(metadata, costCalculator, session, rule, transactionManager, accessControl);
     }
 
+    public RuleAssert assertThat(RuleSet rules)
```

Comment:
put it close to the other `assertThat` and use that other `assertThat` in implementation of this one